astropy/modeling/functional_models.py

```diff
@@ -274,8 +274,8 @@ class Gaussian2D(Fittable2DModel):
 
     Notes
     -----
-    Either all or none of input ``x, y``, ``[x,y]_mean`` and ``[x,y]_stddev``
-    must be provided consistently with compatible units or as unitless numbers.
+    The ``x, y``, ``[x,y]_mean``, and ``[x/y]_stddev`` inputs must have
```

Comment:
Why change `,` to `/` but only for `stddev` but not `mean`?

Reply:
Fixed, thanks.